lib/license.js

```diff
@@ -153,3 +220,28 @@ function writeTemplatedLicense(newName, oldName, text, pkgName) {
   } catch(_) {
   }
 }
+
+function hasOthers() {
+  var old = readLicense();
+  return old && /strongloop.*other.*contributors/i.test(old);
+}
+
+function hasStrongLoop(dual) {
+  if (dual != null)
+    return dual;
+  var old = readLicense();
+  return old && /strongloop subscription agreement/i.test(old);
+}
+
+
+function readLicense() {
+  return readFile('LICENSE') || readFile('LICENSE.md');
+}
+
+function readFile(name) {
+  try {
+    return fs.readFileSync(name);
+  } catch(er) {
+    return null;
```

Comment:
I really like this kind of wrapper because it allows for a more [confident coding](https://www.youtube.com/watch?v=T8J0j2xJFgQ) style.. but because of JS's type coercion, this should return an empty string instead of `null`. It is equally falsey as required by your use, but it won't coerce to a non-empty string if you append it to a string and it also a `.length` attribute that `null` doesn't so you can avoid even more conditionals.